configuration/multimedia

```diff
@@ -1,136 +1,160 @@
+`cp /usr/share/spotify/icons/spotify-linux-16.png /usr/share/icons/hicolor/16x16/apps/spotify.png`
 
-2) Install libdbdread4 from the default repositories:
-   In Debian Template VM:
-   apt-get install libdvdread4
-   
-3) Install libdvdcss for the decryption of CSS protected-DVDs.
-   Because of license restrictions in various countries this can't be done from the Debian repositories but needs to be installed manually.
-   (not that hard :-)).
-   Download the packaged .deb from: http://www.deb-multimedia.org/dists/stable/main/binary-amd64/package/libdvdcss2
-   In Debian Template VM:
-   wget http://www.deb-multimedia.org/pool/main/libd/libdvdcss/libdvdcss2_1.3.0-dmo1_amd64.deb
-   sudo dpkg -i libdvdcss2_1.3.0-dmo1_amd64.deb
 
-4) I like to use VLC to playback Video/DVDs - you might want to use another player, but VLC works fine.
-   As this is not installed in the default Debian Template:
-   In Debian Template VM:
-   sudo apt-get install vlc
-   This will also install some dependencies.
+Installation of VLC
+-------------------
 
-5) Shutdown the Debian Template VM and launch your multimedia App VM
+To play DVDs you can install VLC with the needed Codecs
 
-6) Plugin your external USB DVD-Drive (if you don't have an internal drive) and pass it to the multimedia App VM
-   In dom0:
-   qvm-usb -a multimedia sys-usb:3-6
+Add Repository for libdvdcss
+(See also: http://www.videolan.org/developers/libdvdcss.html)
 
-7) Launch VLC
-   qvm-run multimedia vlc
-   Warning: a window will open in the background which will ask for collecting metadata information.
-   I haven't seen the window as it was covered by the VLC main window, so that I couldn't launch any commands in this window.
+Add GPG-Key
+`apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 8F0845FE77B16294429A79346BCA5E4DB84288D9`
 
-8) Open DVD within VLC:
-   Media > Open Disk
-   You can leave the defaults (/dev/sr0) and just hit play
+> Public-Key: http://keyserver.ubuntu.com/pks/lookup?op=vindex&search=0xB84288D9&fingerprint=on
+> Fingerprint: 8F08 45FE 77B1 6294 429A  7934 6BCA 5E4D B842 88D9 
 
-Enjoy your DVD
+Add repositories to install VLC and libdvdcss
+`echo "deb http://download.videolan.org/pub/debian/stable/ /" >> /etc/apt/sources.list`
+`echo "deb-src http://download.videolan.org/pub/debian/stable/ /" >> /etc/apt/sources.list`
+
+Update package repositories
+`apt-get update`
+
+Install libdvdcss and VLC
+`apt-get install -y libdvdcss2 vlc` 
+
+
+ 
+Installation Google Chrome
+--------------------------
+
+To play Videos with Netflix, Amazon Prime & Co using Chrome is a good option as it has all needed codecs included.
+Hint: Using Chromium will not work for some reasons.
+
+
+Download Google Chrome package from the Google Debian repository
+
+`wget -c https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb`
+
+FIXME: Howto verify the debian package or better adding a Google Debian package repository?
```

Comment:
Not sure about Debian, but I think our Fedora TemplateVM already has the appropriate repo. You just have to enable it. (Similarly for RPMFusion, IIRC.)

Reply:
Ok, I found out that the signing keys can be downloaded here: https://www.google.com/linuxrepositories/

> Key Details
> Download: https://dl.google.com/linux/linux_signing_key.pub
> Key ID: Google, Inc. Linux Package Signing Key <linux-packages-keymaster@google.com>
> Fingerprint: 4CCA 1EAF 950C EE4A B839 76DC A040 830F 7FAC 5991
> Google, Inc. (Linux Package Signing Authority) <linux-packages-keymaster@google.com>
> Fingerprint: EB4C 1BFD 4F04 2F6D DDCC EC91 7721 F63B D38B 4796

to verify the downloaded package:

> On an RPM-based system (Fedora, SUSE, Mandriva, RHEL, etc.),
> download the key and then use rpm to install it.
> wget https://dl.google.com/linux/linux_signing_key.pub
> sudo rpm --import linux_signing_key.pub
> You can verify the key installation by running:
> rpm -qi gpg-pubkey-7fac5991-*
> To manually verify an RPM package, you can run the command:
> rpm --checksig -v packagename.rpm

Reply:
But this shouldn't be necessary if your TemplateVM already has the repo file and key.

Reply:
In Debian template we don't have this repo installed, so this step is needed. Instruction should include adding repository + key and only then installing something from there. Instead of installing deb downloaded from the network.